README.md

```diff
@@ -30,9 +30,9 @@ Some guidelines for contribution:
 - Mind your audience: If you're contributing beginner-oriented content, try to keep things as simple as possible,
   linking to more advanced topics if needed.
 - Try to maintain consistent style. Some very rough points to touch on:
-  - Indent with 4 spaces
-  - Braces on the same line
-  - Pointers/references on the left
+  - Indent with one tab character
```

Comment:
Very nice try. You see, another reviewer might have missed this and merged it but I have eagle eyes and you'll never sneak anything past me!

Reply:
i wasn't trying to sneak anything, "update" means things are changed